CalculatorLibrary/Calculation.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace CalculatorLibrary;
+
+public class Calculation
+{
+    protected double num1 = 0;
+    protected double num2 = 0;
+    public double Result { get; protected set; } = 0;
+    protected string operatorSymbol = "";
+
+    public Calculation(double num1, double num2, double result, string op)
+    {
+        this.num1 = num1;
+        this.num2 = num2;
+        Result = result;
+        this.operatorSymbol = PrettifyOperator(op);
```

Comment:
Whenever you see faded code like this, hover over it and follow the compiler's suggestions. Usually they help you write less code: 
![image](https://user-images.githubusercontent.com/108711476/203435466-07e1ee47-870c-40d5-81ca-ded265d7666c.png)